zulip_bots/setup.py

```diff
@@ -3,14 +3,27 @@
 
 from __future__ import print_function
 
+import itertools
+import os
 import sys
 
 import generate_manifest
 
+def version():
+    # type: () -> str
+    version_py = os.path.join(os.path.dirname(__file__), "zulip_bots", "__init__.py")
+    with open(version_py) as in_handle:
+        version_line = next(itertools.dropwhile(lambda x: not x.startswith("__version__"),
+                                                in_handle))
+    version = version_line.split('=')[-1].strip().replace('"', '')
+    return version
```

Comment:
why aren't you just importing the file?

Reply:
Well, no reason, except that in `zulip/__init__.py`, we do the exact same thing to find the version, so I assumed that there might be a reason behind why we had opted to go that route so I wanted to do it the same way in the other ones. I'm guessing there is no good reason to stick with this verbose approach, so I'll just import it. :) Thanks!

Reply:
I think that's because in that package, we need the dependencies installed to import `__init__.py`, aka the main module, and that is needed to import `zulip.version`.  I suspect a more elegant fix would be to use a symlink to it in the `setup.py` directory and having `setup.py` import it from that path.

Reply:
Awesome, I'll do that! Thanks! :)